src/main/scala/org/clulab/wm/eidos/groundings/grounders/SRLCompositionalGrounder.scala

```diff
@@ -273,7 +386,8 @@ class SRLCompositionalGrounder(name: String, domainOntology: DomainOntology, w2v
       GroundedSpan(trimmedChunk, propertyOpt.get, isProperty = true)
     } else {
       // Otherwise, ground as either a process or concept
-      GroundedSpan(trimmedChunk, groundToBranches(Seq(CONCEPT, PROCESS), trimmedChunk, s.sentence, topN, threshold), isProperty = false)
+      GroundedSpan(trimmedChunk, groundToBranches(Seq(CONCEPT, ENTITY, PROCESS), trimmedChunk, s.sentence, topN,
```

Comment:
This is nitpicky, but I see several of these Seqs around.  Could they be turned into variables with explanatory names and added to the companion object?  They are being recreated repeatedly.

Reply:
+1 for this

Reply:
Good idea! I added some variables in the object and replaced the various Seqs above.